pkg/util/chunk/chunk_util.go

```diff
@@ -112,7 +112,7 @@ func CopySelectedRows(dstCol *Column, srcCol *Column, selected []bool) {
 func CopySelectedRowsWithRowIDFunc(dstCol *Column, srcCol *Column, selected []bool, start int, end int, rowIDFunc func(int) int) {
 	if srcCol.isFixed() {
 		for i := start; i < end; i++ {
```

Comment:
Should we judge ```selected !=nil``` and then jump into ```for i := start; i < end; i++ ```

Reply:
> Should we judge `selected !=nil` and then jump into `for i := start; i < end; i++ `

okk, I make the function returns at the beginning when `selected == nil`

Reply:
> Should we judge `selected !=nil` and then jump into `for i := start; i < end; i++ `

I create a new function named `CopyAllRowsWithRowIDFunc` for the task that copies all rows.